### ISSUE TITLE
AWS Backup Code to copy snapshot to different region

### ISSUE THREAD

**Issue by infa-sgowtham** [terraform-providers/terraform-provider-aws#12306]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

We have a feature in AWS Backup Service to copy the snapshots to a different region. But, as of now, we have terraform code only to backup on the current environment. If will be great if we have the feature to copy the backup to the different regions also. I am referring to the below documentation page "https://www.terraform.io/docs/providers/aws/r/backup_plan.html#lifecycle-arguments" Or if that feature is already present on terraform, please guide me to find the suitable code.

### New or Affected Resource(s)

* aws_XXXXX

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by ewbankkit** [comment]
@infa-sgowtham Thanks for this.
This functionality has already been requested in https://github.com/terraform-providers/terraform-provider-aws/issues/11589 and is addressed via https://github.com/terraform-providers/terraform-provider-aws/pull/11923.

**Comment by infa-sgowtham** [comment]
@ewbankkit Thanks for your response. I don't see this functionality on the master branch yet. Do you have any idea when it will live?

**Comment by infa-sgowtham** [comment]
@ewbankkit  Do you have any update when this feature will be live?

**Comment by DrFaust92** [comment]
Hey @infa-sgowtham, support for this was add in 2.58 of the provider
closed via #11923
